.circleci/config.yml

```diff
@@ -1099,6 +1099,10 @@ workflows:
             - b_bytecode_osx
             - b_bytecode_ems
 
+      # Code Coverage enabled build and tests
+      - b_ubu_codecov: *workflow_trigger_on_tags
+      - t_ubu_codecov: *workflow_ubuntu2004_codecov
```

Comment:
`t_ubu_codecov` runs only syntax tests. This means that SMTchecker and external tools like `yul-phaser` and `solidity-upgrade` count as lacking any coverage even if they do have tests.

You should at least add a run with `--run_test=Phaser` - this will execute only phaser's tests suite and it should be pretty well covered, it has tons of unit tests. But maybe we should just run the whole suite (including command-line tests) to be sure everything is included.

Alternatively, we could exclude directories that definitely won't be covered by syntax tests alone.

Reply:
Oh, actually I see that it does run the whole suite (via `run_soltest_all`).

But in https://app.codecov.io/gh/ethereum/solidity/branch/codecov which you posted earlier phaser is all red. Was that a result of a syntax-test-only run? I think that tools and SMTchecker code should be excluded from that run.